crates/sui-adapter-transactional-tests/tests/mvcc/receive_object_dof.exp

```diff
@@ -6,107 +6,107 @@ A: object(0,0)
+Contents: sui::dynamic_field::Field<sui::dynamic_object_field::Wrapper<u64>, sui::object::ID> {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,0)}}, name: sui::dynamic_object_field::Wrapper<u64> {name: 0u64}, value: sui::object::ID {bytes: fake(2,3)}}
 
-task 10 'view-object'. lines 80-80:
+task 9 'view-object'. lines 80-80:
 Owner: Object ID: ( fake(7,0) )
 Version: 3
-Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,2)}}, value: 0u64}
+Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,1)}}, value: 0u64}
 
-task 11 'view-object'. lines 82-82:
+task 10 'view-object'. lines 82-82:
 Owner: Account Address ( A )
 Version: 3
+Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,2)}}, value: 0u64}
+
+task 11 'view-object'. lines 84-84:
+Owner: Object ID: ( fake(2,0) )
+Version: 2
 Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,3)}}, value: 0u64}
 
-task 12 'programmable'. lines 84-85:
+task 12 'programmable'. lines 86-87:
 mutated: object(0,0), object(2,1), object(2,2), object(2,3)
 gas summary: computation_cost: 1000000, storage_cost: 4818400,  storage_rebate: 4770216, non_refundable_storage_fee: 48184
 
-task 13 'view-object'. lines 87-89:
-Owner: Object ID: ( fake(2,2) )
+task 13 'view-object'. lines 89-91:
+Owner: Object ID: ( fake(2,1) )
 Version: 2
-Contents: sui::dynamic_field::Field<sui::dynamic_object_field::Wrapper<u64>, sui::object::ID> {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,0)}}, name: sui::dynamic_object_field::Wrapper<u64> {name: 0u64}, value: sui::object::ID {bytes: fake(2,1)}}
-
-task 14 'view-object'. lines 90-90:
-Owner: Object ID: ( fake(2,0) )
-Version: 4
-Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,1)}}, value: 3u64}
+Contents: sui::dynamic_field::Field<sui::dynamic_object_field::Wrapper<u64>, sui::object::ID> {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,0)}}, name: sui::dynamic_object_field::Wrapper<u64> {name: 0u64}, value: sui::object::ID {bytes: fake(2,3)}}
 
-task 15 'view-object'. lines 92-92:
+task 14 'view-object'. lines 92-92:
 Owner: Object ID: ( fake(7,0) )
 Version: 4
-Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,2)}}, value: 2u64}
+Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,1)}}, value: 2u64}
 
-task 16 'view-object'. lines 94-94:
+task 15 'view-object'. lines 94-94:
 Owner: Account Address ( A )
 Version: 4
-Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,3)}}, value: 1u64}
+Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,2)}}, value: 1u64}
+
+task 16 'view-object'. lines 96-96:
+Owner: Object ID: ( fake(2,0) )
+Version: 4
+Contents: tto::M1::A {id: sui::object::UID {id: sui::object::ID {bytes: fake(2,3)}}, value: 3u64}
```

Comment:
I think this is just because `(2,0)` and `(2,3)` swapped places. The diff above looks like this one now. I'm happy to investigate more if that isn't the case, however.